be/src/vec/functions/function_helpers.cpp

```diff
@@ -26,6 +26,42 @@
 
 namespace doris::vectorized {
 
+Block create_block_with_nested_columns_only_args(const Block& block, const ColumnNumbers& args) {
+    std::unordered_set<size_t> args_set(args.begin(), args.end());
+    Block res;
+    size_t columns = block.columns();
+
+    for (size_t i = 0; i < columns; ++i) {
```

Comment:
only process columns in args_set instead of all columns

Reply:
ok